resources/theaters/germanycw/info.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: GermanyCW # Mostly based on Caucasus, could be updated with more appropriate data if necessary
+timezone: +1
+daytime:
+  dawn: [6, 9]
+  day: [9, 18]
+  dusk: [18, 20]
+  night: [0, 5]
+climate:
+  day_night_temperature_difference: 9.0
+  seasons:
+    winter:
+      average_pressure: 29.72 # TODO: Find real-world data
```

Comment:
This might be a good place to start: https://www.dwd.de/EN/ourservices/cdc/cdc_ueberblick-klimadaten_en.html

Reply:
Let's deal with this later...